client/lib/interval/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { PropTypes } from 'react';
+
+/**
+ * Calls a given function on a given interval
+ */
+export default React.createClass( {
+	displayName: 'Interval',
+
+	propTypes: {
+		onTick: PropTypes.func.isRequired,
+		milliseconds: PropTypes.number,
+		leading: PropTypes.bool,
+		pauseWhenHidden: PropTypes.bool,
+		children: PropTypes.element
+	},
+
+	getDefaultProps: () => ( {
```

Comment:
If I were to use `this` in this method I think it would probably be undefined or maybe reference `window` but definitely something other than the expected `React` object.

The fat arrow here could cause confusion.